conda/install.py

```diff
@@ -227,17 +227,20 @@ def exp_backoff_fn(fn, *args):
     """Mostly for retrying file operations that fail on Windows due to virus scanners"""
     if not on_win:
         return fn(*args)
-    import random
 
+    import random
     max_retries = 5
     for n in range(max_retries):
         try:
             result = fn(*args)
-        except Exception as e:
+        except EnvironmentError as e:
             log.debug(repr(e))
-            if n == max_retries-1:
+            if e.errno in (errno.EPERM, errno.EACCES):
+                if n == max_retries-1:
+                    raise
+                time.sleep(((2 ** n) + random.random()) * 1e-3)
```

Comment:
Here I'd actually like, when debugging, enough time to go to the file in question using Unlocker to see what's got it open, perhaps we can have it in seconds for debug?